### PR TITLE
fix(mocks): adjust token mock

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## Description

Adjust `token.ts` mocks with a mocked jwt token, to get rid of following error:
`ERROR DOMException: Failed to execute 'atob' on 'Window': The string to be decoded is not correctly encoded.` 

- Token is generated from `permisson.ts`

image

### CODE REVIEW

**Review by dzhavat — DISMISSED** [review]

**Review by dzhavat — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "@popperjs/core": "^2.11.6",
     "@rx-angular/cdk": "1.0.0-rc.3",
     "@rx-angular/template": "1.0.0-rc.4",
+    "@types/crypto-js": "4.1.1",
```
[the remark this answers is not in the document]
Comment (frontend-specialisten) [reply package.json:77]: Problem was that it requires to polyfill `crypto` and `stream` which I didn't have much luck doing. Then I found this solution and noticed `crypto-js` already was part of our dependencies (however it should properly be added to package.json, to make that visible).

[the remark this answers is not in the document]
Comment (frontend-specialisten) [reply package.json]: I've added `crypto-js` and moved `@types/crypto-js` to dev-dependencies

File: libs/dh/shared/domain/src/lib/permission.ts [hunk 2]
```diff
@@ -14,9 +14,10 @@
...
-  | 'gridareas:manage'
-  | 'actor:manage'
-  | 'users:manage';
+export const permissions = [
+  'organization:manage',
+  'gridareas:manage',
+  'actor:manage',
+  'users:manage',
+] as const;
+export type Permission = typeof permissions[number];
```
Comment (dzhavat) [inline libs/dh/shared/domain/src/lib/permission.ts:23]: praise: nice change.

question: not that it should be done, but could this also be in a `domain` library under `dh/auth`?

**Review by dzhavat — APPROVED** [review]

File: libs/dh/shared/domain/src/lib/permission.ts (hunk 2 above)
Comment (frontend-specialisten) [reply libs/dh/shared/domain/src/lib/permission.ts:23]: I don’t see anything should be wrong doing that at a later point :)

**Pull request merged**